pkg/addon-operator/operator.go

```diff
@@ -1326,46 +1362,56 @@ func (op *AddonOperator) HandleModuleHookRun(t sh_task.Task, labels map[string]s
 		}
 	}
 
-	// Module hook can recreate helm objects, so pause resources monitor.
-	// Parallel hooks can interfere, so pause-resume only for hooks in the main queue.
-	// FIXME pause-resume for parallel hooks
-	if t.GetQueueName() == "main" {
-		op.HelmResourcesManager.PauseMonitor(hm.ModuleName)
-		defer op.HelmResourcesManager.ResumeMonitor(hm.ModuleName)
-	}
+	if shouldRunHook {
+		logEntry.Infof("Module hook start '%s'", hm.HookName)
 
-	err = op.ModuleManager.RunModuleHook(hm.HookName, hm.BindingType, hm.BindingContext, t.GetLogLabels())
+		// Module hook can recreate helm objects, so pause resources monitor.
+		// Parallel hooks can interfere, so pause-resume only for hooks in the main queue.
+		// FIXME pause-resume for parallel hooks
+		if t.GetQueueName() == "main" {
```

Comment:
It is not connected to the PR subject, but I just curious about setting `queue: main` in a hook config. Is it allowed? Don't we want to forbid explicitly setting queue to "main"?

Reply:
A good rule for future linting. Also, we can use something different instead of the "main" string.